link_resolver_twitter.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+    "bytes"
+    "encoding/json"
+    "errors"
+    "fmt"
+    "net/http"
+    "os"
+    "time"
+
+    "log"
+    "net/url"
+    "regexp"
+    "strings"
+    "text/template"
+)
+
+const tweeterTooltip = `<div style="text-align: left;">
+<b>{{.Text}}</b>
+<br><b>Author:</b> {{.Username}}
+</div>
+`
+
+var (
+    tweetRegexp = regexp.MustCompile(`(?i)\/.*\/status(?:es)?\/([^\/\?]+)`)
+)
+
+type TweetApiResponse struct {
+    Data []struct {
+        ID          string `json:"id"`
+        AuthorID    string `json:"author_id"`
+        Text        string `json:"text"`
+        Attachments struct {
+            MediaKeys []string `json:"media_keys"`
+        } `json:"attachments"`
+    } `json:"data"`
+    Includes struct {
+        Users []struct {
+            Name     string `json:"name"`
+            ID       string `json:"id"`
+            Username string `json:"username"`
+        } `json:"users"`
+    } `json:"includes"`
+    Errors []struct {
+        Detail       string `json:"detail"`
+        Title        string `json:"title"`
+        ResourceType string `json:"resource_type"`
+        Parameter    string `json:"parameter"`
+        Value        string `json:"value"`
+        Type         string `json:"type"`
+    } `json:"errors"`
+}
+
+type tweetTooltipData struct {
+    Text      string
+    Username  string
+    Thumbnail string
+}
+
+func init() {
+    bearerKey, exists := os.LookupEnv("CHATTERINO_TWITTER_BEARER_KEY")
```

Comment:
Rename the environment variable to `CHATTERINO_API_TWITTER_BEARER_TOKEN`

Reply:
fixed